cmd/release/cmd/root.go

```diff
@@ -21,9 +28,9 @@ var rootCmd = &cobra.Command{
 // Execute adds all child commands to the root command and sets flags appropriately.
 // This is called by main.main(). It only needs to happen once to the rootCmd.
 func Execute() {
+	cobra.OnInitialize(initConfig)
 	if err := rootCmd.Execute(); err != nil {
-		fmt.Println("error: " + err.Error())
-		os.Exit(1)
+		log.Fatal(err)
```

Comment:
Will this add additional info to the output? If so I'd prefer we keep the output only what we need.

Reply:
Updated, `rootCmd.Execute()` already prints the error, printing it again duplicates the output